File: q/c/submissions/nothingreallymatters/problem1.c
#include<stdio.h>
int main()
{
 long n;
 int flag=0;
 scanf("%d",&n);
 for(int i=2;i<=(n/2);i++)
 {
  if((n%i)==0)
    flag=1;
 }
 if(flag==1)
   printf("No");
 else if(flag==0)
   printf("Yes");
 return 0;
}

File: q/c/submissions/Stephenites/problem1.java
import java.util.*;
public class problem1
{
public static void main(String args[])
{
Scanner in=new Scanner (System.in);
int i,n,m,flag;
n=in.nextInt();
m=n/2;
if(n==0||n==1)
{
System.out.println("No");
}
else
{
for(i=2;i<=m;i++)
{
if(n%i==0)
System.out.println("yes");
flag=1;
}
}
}
}
